lib/node_modules/@stdlib/blas/ext/base/sapxsum/docs/repl.txt

```diff
@@ -85,14 +82,13 @@
     Examples
     --------
     // Standard Usage:
-    > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 2.0 ] );
+    > x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 2.0 ] );
     > {{alias}}.ndarray( x.length, 5.0, x, 1, 0 )
     16.0
 
     // Using offset parameter:
-    > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0 ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > {{alias}}.ndarray( N, 5.0, x, 2, 1 )
+    > x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0 ] );
```

Comment:
same comment here

Reply:
Since it is already declared in the first example, it must not be re-declared in the second one, right ? Or am I missing out on something here @Pranavchiku ?